go.mod

```diff
@@ -1,11 +1,12 @@
 module github.com/flightctl/flightctl
 
-go 1.22
+go 1.22.0
+
+toolchain go1.23.7
```

Comment:
Please don't bump up the Golang version, we're relying on specific versions for the up- and downstream builds.